### PR TITLE
fix(connect-iframe): change naming more meaninful

### PR DESCRIPTION
**Pull request by karliatto**
## Description
Changing `whitelistedMessages` to `blockedMessages` and `targetUiEvent` to `isTargetUiEvent`.

### CODE REVIEW

File: packages/connect-iframe/src/index.ts [hunk 1]
```diff
@@ -177,7 +177,7 @@ const targetUiEvent = (message: CoreMessage) => {
         UI.BUNDLE_PROGRESS,
         UI.ADDRESS_VALIDATION,
     ];
-    return message.event === UI_EVENT && whitelistedMessages.indexOf(message.type) < 0;
+    return message.event === UI_EVENT && blockedMessages.indexOf(message.type) < 0;
```
Comment (szymonlesisz) [inline packages/connect-iframe/src/index.ts]: i think that those are the **only** messages that are allowed to be sent to 3rd party window, therefore they are whitelisted

if UI_EVENT message type **is not** whitelisted *[you can change it to .includes(message.type) btw] then the message should be sent to `_popupMessagePort` (for example UI.REQUEST_PIN)

so i think function name should be changed to `shouldUiEventBeSentToPopup` :)

Comment (mroz22) [reply packages/connect-iframe/src/index.ts]: `shouldUiEventBeSentToPopup` sounds quite descriptive, I'd do that. We were wrapping our head around these files and I guess we might come up with couple of more improvements in naming that would make them more readable

Comment (karliatto) [reply packages/connect-iframe/src/index.ts]: What about this changes ed19cdebcb5066b5cbc25c8e20f3e50df0d51782 ?

Comment (mroz22) [reply packages/connect-iframe/src/index.ts]: ok with me

**Review by mroz22 — APPROVED** [review]

**Pull request merged**